### PR TITLE
Refer to topics by content_id instead of slug

### PR DESCRIPTION
**Pull request by tijmenb**
When tagging a document in Whitehall, we save the slug of the topic (or specialist sector) in the `SpecialistSector` model.

We'd like to persist the `content_id` instead. This will:

- Make the tagging form work like content-tagger & policy-publisher
- Could allow us to change the URLs of topics (a long standing request)
- Removes the need for a content_id lookup before sending a content item to the publishing-api

https://trello.com/c/QYgPe6kn

### CODE REVIEW

**Comment by tijmenb** [comment]
Closing this to not interfere with the Detailed Guides migration. @gpeng will give a shout when that's done and this can be picked up again.

**Comment by tijmenb** [comment]
@gpeng what's the status? can we start this again?

**Comment by gpeng** [comment]
We're currently blocked on [this](https://github.com/alphagov/whitehall/pull/2747) which I think looks good to merge now and then we just need to do a small bit of work and switch them over to government-frontend with a republish. So... not quite yet but very soon.

File: db/migrate/20161010135926_add_content_id_to_specialist_sector.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
..., "/topic/mot/manuals"=>"71e36c84-c058-470b-8790-f4c8fd39c615", "/topic/driving-motorcyle-instructors/approved-driving-instructors"=>"bb85c76f-b660-4426-a1c3-37313a2cea81", "/topic/driving-tests-and-learning-to-drive/motorcycle"=>"cb82959c-e637-46aa-8cae-6475ee56b9f2", "/topic/mot/run-mot-test-station"=>"92bb4c52-8c01-463d-a5e1-b841783eff28", "/topic/mot/lorry-bus-trailer-annual-test"=>"2144a80d-3cb0-472c-a0a9-ae738004acf9", "/topic/schools-colleges-childrens-services/opening-academy-free-school"=>"5b8c90ad-acbd-48c5-a39d-af5b841bcd5a", "/topic/business-tax/self-employed"=>"9741abcb-c246-4fd6-bff3-7cee244a9979", "/topic/intellectual-property/copyright"=>"2db5f38b-cf66-455a-bd43-ca4d7c8289df", "/topic/intellectual-property/patents"=>"a98b1005-f153-4012-95cd-1d513b7d28c6", "/topic/intellectual-property/trade-marks"=>"c8e6f8e2-4dac-4ce3-bb02-8690b0e284e4", "/topic/intellectual-property/designs"=>"a949350d-98ed-4260-beb9-f2fc69d15946", "/topic/local-government/byelaws"=>"b01b8809-1fe1-43ff-be0b-8c2581ce6a70", "/topic/driving-motorcyle-instructors/large-goods-vehicle-instructors"=>"5dd90066-ad4b-45c0-b007-cf5bd8cf462e", "/topic/mot/modernisation"=>"020da934-e98c-4925-8e3d-b4fae3fc85f6"}
```
Comment (gpeng) [inline db/migrate/20161010135926_add_content_id_to_specialist_sector.rb]: Can we split this line?

Comment (gpeng) [inline db/migrate/20161010135926_add_content_id_to_specialist_sector.rb]: Discussed offline... This migration will be removed in a subsequent commit so this is fine.

**Pull request merged**